cypress/integration/Stepper._spec.js

```diff
@@ -134,11 +134,11 @@ describe("Stepper.vue", () => {
     });
 
     it("Can go back by clicking on Previous button", () => {
-        cy.visit("http://localhost:8080/ ");
+        cy.visit("/");
         cy.get(".step-0")
             .find("[value=yes]")
             .check();
-        cy.get(".step-navigation > .button").click();
+        cy.get(".is-success").click();
```

Comment:
Please use `.next-button` class for Next button, and `.previous-button` for 'Previous' button everywhere.